bindings/python/iota_sdk/client/_high_level_api.py

```diff
@@ -80,31 +80,35 @@ def _call_method(self, name, data=None):
         no payload.
         """
 
+    # TODO: this should return `List[OutputResponse]`, not `List[OutputWithMetadata]`
+    # https://github.com/iotaledger/iota-sdk/issues/1921
```

Comment:
Isn't that something that should be done in this PR? Rust already return OutputResponse?

Reply:
The core binding doesn't use Rust's `get_outputs`, but `get_outputs_with_metadata`.

Reply:
Will be fixed in #1928.